### DISCUSSION TITLE
Multimer protein

### DISCUSSION THREAD

**Discussion by Vini161** [marrink-lab/vermouth-martinize#638] [Q&A]
I am trying martini with go bias, with the webserver recommended: http://pomalab.ippt.pan.pl/GoContactMap/ . It gives 2 contact map output- go_contacts.txt and contact_map.out.  The itp i get after using martinize is a single molecule0.itp. Will i need seperate chain itp for martini go like model? Do the inter and intra monomer potentials get defined by default?

**Comment by csbrasnett** [comment]
If you're using the latest version of martinize2, then you can use the go model by specifying `-go contact_map.out`, which will generate the contacts found from the file you gave to the server. There's some more documentation for the Go model options [here](https://vermouth-martinize.readthedocs.io/en/latest/tutorials/go_models.html) but usually the defaults should work fine.

For multidomain proteins you may want to consider biasing any linking domains using the `-water-bias` flags (some more details [here](https://vermouth-martinize.readthedocs.io/en/latest/tutorials/water_biasing.html)), which are detailed in the [paper](https://www.biorxiv.org/content/10.1101/2024.04.15.589479v1) describing the Go model.

**Reply by Vini161** [reply]
Thank you! I am currently trying the latest Go-like model on a dimer. Is water-biasing mandatory for the simulation, or will the regular Go-like model suffice?

**Reply by csbrasnett** [reply]
That's likely to be highly system dependent, I'd recommend reading the paper I linked to understand more about the factors that make water-biasing necessary. In general for soluble proteins with disordered linking domains I'd recommend using the water biasing as described though.

**Reply by Vini161** [reply]
Thanks! I went through it and it is indeed relevant for my system, I did
martinize2 -f dimer.pdb -o topol.top -x cg_dimer.pdb -dssp -id-regions 1:26 362:388 -idr-tune -water-bias -water-bias-eps idr:0.5

I saw this in go_nbparams.itp
W molecule_0_1 1 0.47000000 0.50000000 ;water bias C
W molecule_0_2 1 0.47000000 0.50000000 ;water bias C
W molecule_0_3 1 0.41000000 0.50000000 ;water bias C
W molecule_0_4 1 0.41000000 0.50000000 ;water bias S
W molecule_0_5 1 0.47000000 0.50000000 ;water bias C
W molecule_0_6 1 0.47000000 0.50000000 ;water bias C
W molecule_0_7 1 0.47000000 0.50000000 ;water bias C
W molecule_0_8 1 0.47000000 0.50000000 ;water bias C
W molecule_0_9 1 0.41000000 0.50000000 ;water bias S
W molecule_0_10 1 0.41000000 0.50000000 ;water bias S
W molecule_0_11 1 0.47000000 0.50000000 ;water bias C
W molecule_0_12 1 0.47000000 0.50000000 ;water bias C
W molecule_0_13 1 0.41000000 0.50000000 ;water bias C
W molecule_0_14 1 0.41000000 0.50000000 ;water bias C
W molecule_0_15 1 0.41000000 0.50000000 ;water bias C
W molecule_0_16 1 0.41000000 0.50000000 ;water bias E
W molecule_0_17 1 0.47000000 0.50000000 ;water bias E


1.	Just to clarify, it initially recognized residue 1 as C, but now it is treating it as water-biased after I defined it as IDR. Is that correct?
2. Is 0.5 (as suggested by tutorial) an acceptable value?
